portfolio/src/main/webapp/chart.html

```diff
@@ -5,8 +5,9 @@
   <script src="https://www.gstatic.com/charts/loader.js"></script>
   <script src="script.js"></script>
 </head>
-<body onload="loadChartsApi()">
+<body onload="loadChartsApi(); loadChartsApi2();">
```

Comment:
I think rather than having 2 functions that each call `google.charts.load`, it might be better to call `google.charts.load` only once and request both chart types.  You could instead do something like this:
```
function loadChartsApi2() {
  google.charts.load('current', {'packages':['corechart', 'line']});
  google.charts.setOnLoadCallback(drawChart);
  google.charts.setOnLoadCallback(drawChart2);
}
```

I think that right now since you are calling `google.charts.load` twice, you may also be calling each of your drawChart functions twice.  Also doing `google.charts.load` just once should have fewer network requests.

Reply:
addressed in "optimize loadChartsApi function"